applications/dds/dds_video/README.md

```diff
@@ -23,8 +23,8 @@ Holoviz output.
 
 ## Prerequisites
 
-- This application requires [RTI Connext](https://content.rti.com/l/983311/2024-04-30/pz1wms)
-be installed and configured with a valid RTI Connext license prior to use. 
+- This application requires an installation of [RTI Connext Express](https://content.rti.com/l/983311/2025-07-08/q5x1n8) to provide access to the DDS domain.
+To obtain a license/activation key, please [click here](https://content.rti.com/l/983311/2025-07-25/q6729c). Please see the [usage rules](https://www.rti.com/products/connext-express) for Connext Express.
```

Comment:
> This application requires an installation of [RTI Connext Express](https://content.rti.com/l/983311/2025-07-08/q5x1n8)

This takes me to the trial page rather than Connext Express, is that expected? Could we update to https://www.rti.com/products/connext-express

> To obtain a license/activation key, please [click here](https://content.rti.com/l/983311/2025-07-25/q6729c)

I was surprised that this immediately started a download, seems like RTI typically wants new users to register, should we instead redirect to the free trial registration at https://www.rti.com/free-trial ?